wait_for_db.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo "Wait db"
+while ! nc -z $POSTGRES_DB $POSTGRES_PORT; do
```

Comment:
The `nc` command is using `$POSTGRES_DB` as the host, which is incorrect. It should be `$POSTGRES_HOST` to correctly refer to the database host. Ensure that the environment variable for the host is set correctly.